src/components/common/calendar/Calendar.js

```diff
@@ -30,16 +30,16 @@ function CalendarApp() {
   // TO-DO: Implement Auth0
   useEffect(() => {
     if (eventFlag) {
-      //TO-DO: Implement axiosWithAuth once we've adjusted it to work with Auth0
-      axios
-        .get('/calendar-events/user')
+      // axiosWithAuth(idToken)
+      //   .get('/calendar-events/user')
+      Promise.resolve({ data: { events: [] }, message: '' })
```

Comment:
Great job on the promises, this will be great to work with until the backend is up!